src/metabase/api/permissions.clj

```diff
@@ -82,14 +82,11 @@
 
 (api/defendpoint GET "/group"
   "Fetch all `PermissionsGroups`, including a count of the number of `:members` in that group."
-  [limit offset]
-  {
-   limit  (s/maybe su/IntStringGreaterThanZero)
-   offset (s/maybe su/IntStringGreaterThanOrEqualToZero)
-  }
+  []
+  {offset-paging/*limit*  (s/maybe su/IntGreaterThanZero)
+   offset-paging/*offset* (s/maybe su/IntGreaterThanOrEqualToZero)}
```

Comment:
I don't think you should be sticking the dynamic variables in the schema validation map here... I'm surprised it doesn't break anything. 

If we do it here, we should probably do it everywhere, which defeats the purpose of the middleware in the first place -- not having to do it everywhere. I think mentioning that the endpoint supports `offset` and `limit` in the docstring is sufficient.